app-modules/knowledge-base/resources/views/filament/pages/badges.blade.php

```diff
@@ -33,7 +33,36 @@
 --}}
 
 <div class="flex gap-2">
-    <x-filament::badge>
-        {{ $getRecord()?->status?->name ?? '' }}
-    </x-filament::badge>
+    @if ($getRecord()->status)
+        <x-filament::badge>
+            {{ $getRecord()?->status?->name ?? '' }}
```

Comment:
Because you are checking whether or not `$getRecord()->status` is truthy above you can probably simplify this:

```suggestion
            {{ $getRecord()->status->name }}
```

Reply:
Done